.github/workflows/helm.yaml

```
@@ -72,4 +72,5 @@ jobs:
         uses: helm/chart-releaser-action@v1.5.0
         env:
           CR_SKIP_EXISTING: "false"
+          config: .github/chart-releaser-config.yaml
           CR_TOKEN: "${{ secrets.GITHUB_TOKEN }}"
```

```suggestion
          config: .github/chart-releaser-config.yaml
          CR_SKIP_EXISTING: "false"
          CR_TOKEN: "${{ secrets.GITHUB_TOKEN }}"
```

Minor style suggestion